packages/matter-node.js/src/codec/BitmapSchema.ts

```diff
@@ -12,7 +12,7 @@ const enum BitRangeType {
     Enum,
 }
 
-type BitRange<T> = { type: BitRangeType, offset: number, length: number };
+type BitRange<T> = { type: BitRangeType, offset: number, length: number, _type?: T };
```

Comment:
In typescript, when a generic type has a type unused, this triggers a bunch of random bugs in the ts compiler that I painfully debugged in another part of the code in the past.

Basically, since the type is not used, the compiler removes it in some conditions and is not able to correctly track types when the object is used.
Adding an optional "_type: T" doesn't change anything at runtime, but help ts compiler keep track of the typing.

Added a comment to make this clear